src/components/snackbar/themes/snackbar.material.scss

```diff
@@ -0,0 +1 @@
+@use '../../../styles/utilities' as *;
```

Comment:
This file should be scrapped altogether. No need to have an empty file just for the sake of having a material theme.